.travis.yml

```diff
@@ -30,6 +30,7 @@ cache:
   directories:
   - $HOME/.m2
 script:
+  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.2 -Dscala=2.11
   - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.0 -Dscala=2.11
```

Comment:
The original is lack of script for Spark 2.1? @carsonwang

Reply:
Yes. There is a timeout limit on travis so we didn't build for all profiles.